docs/overview.rst

```diff
@@ -47,6 +48,31 @@ For more information, (e.g., connecting to remote nodes, provider auto-detection
 using a test provider) see the :ref:`Providers <providers>` documentation.
 
 
+AsyncProvider
+-------------
+
+- ``AsyncWeb3.AsyncHTTPProvider`` for connecting to http and https based JSON-RPC servers.
+
+.. note::
+
+   The AsyncHTTPProvider is still under active development. Not all JSON-RPC
+   methods and middleware are available yet. The list of available methods and
+   middleware can be seen on the :ref:`AsyncHTTPProvider` docs
+
+.. code-block:: python
+
+   >>> from web3 import AsyncWeb3
+
+   # AsyncHTTPProvider:
+   >>> w3 = AsyncWeb3(AsyncWeb3.AsyncHTTPProvider('http://127.0.0.1:8545'))
+
+   >>> w3.is_connected()
```

Comment:
```suggestion
   >>> await w3.is_connected()
```

Reply:
`await` works if your in ipython. If it's just supposed to be the repl, then need to `import asyncio` and make it:
```suggestion
   >>> asyncio.run (w3.is_connected())
```

Reply:
Nice catch! There is a box that says it's meant to be run in a python interpreter like ipython, so went with just `await`.